packages/rancher-vsphere-csi/charts/templates/vsphere-csi-controller-deployment.yaml

```diff
@@ -25,10 +25,19 @@ spec:
         node-role.kubernetes.io/controlplane: "true"
       {{- end }}
       tolerations:
-        - operator: "Exists"
+        - key: node-role.kubernetes.io/master
+          operator: Exists
```

Comment:
What is the purpose for this change? Before, we were tolerating any NoSchedule or NoExecute, but now we seem to be lessening the scope to only NoSchedule with a specific key, so a user who was relying on this toleration for a custom taint would no longer be able to deploy the chart right?

I think, if we plan to make this change, we need to expose a way to override tolerations.

Also, we need to make sure that the default taints and tolerations for Rancher and RKE2 (e.g. CriticalAddonsOnly) are tolerated by default.

Reply:
This change came from upstream. Nearly everything that isn't templated is a change from upstream. Should we modifying stuff like this from what's upstream, or stick to what's upstream?

Reply:
Reverted change from upstream and kept tolerations as is.